meta/saiserialize.cpp

```diff
@@ -84,6 +84,12 @@ sai_status_t transfer_list(
     {
         transfer_primitive(src_element.count, dst_element.count);
 
+        if (src_element.list == NULL && src_element.count > 0)
```

Comment:
Ah in this case, sure i can move that, anyway this partial override doesen't matter since api will fail